tests/config/align_assign_span-3.cfg

```diff
@@ -0,0 +1 @@
+align_assign_span               = 3
```

Comment:
nit: all your test files are missing a terminal `\n`

Reply:
Well yes, why do you want to have one?

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline ?

Reply:
Indeed :smile:. But to summarize:
- POSIX says so. (So does C, though that is less important for uncrustify's files in `tests/`.)
- Git doesn't like it. (Github sort of doesn't like it either.)
- It messes up various CLI text-processing tools.
- Many editors (of which vim is probably best known) will add the newline, causing unnecessary and "spurious" diff churn.
- Adding lines to the end of a file that is missing a final newline will also cause "spurious" diff churn.
- [Robustness Principle](https://en.wikipedia.org/wiki/Robustness_principle).